EXAMPLES.md

```diff
@@ -254,7 +249,7 @@ module SSHKit
   module Formatter
     class MyFormatter < SSHKit::Formatter::Abstract
       def write(obj)
-        case obj.is_a? SSHKit::Command
+        if obj.is_a? SSHKit::Command
```

Comment:
At least with Ruby 3.2.2, this `case` produced an error:

```
SyntaxError: unexpected `end', expecting `when'
```

Reply:
Good catch!